index.js

```diff
@@ -83,9 +87,15 @@
         }, $this.data('load-after'));
       });
     } else {
-      whenScroll(['within 300px of', $newElement[0]], function () {
-        loadNow($this, $newElement);
-      }, true);
+      // This is essential for firing the event to a DOM immediately, as set by
```

Comment:
I've added you as a collaborator. You know you don't need to be a collaborator to send a PR, right? :P